changelog/snippets/balance.6559.md

```
@@ -0,0 +1,7 @@
+- (#6559) Various fixes and improvements for Megalith's build capabilities:
+    - Fix the build range preview to display the real build range.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)